admin-manual/installation-setup/upgrading/upgrading.rst

```diff
@@ -16,6 +18,36 @@ Upgrade from Archivematica |previous_version|.x to |release|
    While it is possible to upgrade a GitHub-based source install using ansible,
    these instructions do not cover that scenario.
 
+.. _completed-transfers:
+
+Clean up completed transfers watched directory
+----------------------------------------------
+
+Upgrading from Archivematica 1.10.x or older to Archivematica 1.11.x can result
+in a number of completed transfers appearing as failed in the Archivematica
+dashboard, as well as corresponding failure notification emails being sent.
+These are not actual failures, but are unintentional side effects of changes
+made in Archivematica 1.11 to the workflow and to how metadata files are stored
+and copied into the SIP.
+
+To prevent these failures from occuring during an upgrade from Archivematica
+1.10 or lower:
+
+1. Confirm that all transfers and ingests are complete.
+
+Check that there are no transfers or SIPs that are still being processed or
+awaiting decisions in the Transfer and Ingest tabs. If there are, finish
+processing the transfers/ingests before proceeding.
```

Comment:
Indenting inline with the above para will just make this format a bit more nicely.

```suggestion
   Check that there are no transfers or SIPs that are still being processed or
   awaiting decisions in the Transfer and Ingest tabs. If there are, finish
   processing the transfers/ingests before proceeding.
```